src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php

```diff
@@ -938,6 +939,8 @@ public function testObjectNormalizerWithAttributeLoaderAndObjectHasStaticPropert
         $this->assertSame([], $normalizer->normalize($class));
     }
 
+    // accessors
```

Comment:
should be removed

Reply:
It's used to group some other tests in the ObjectNormalizerTest too. 
"skip uninitialized", "type enforcement", etc.

Should it still be removed?